test/Microsoft.Sbom.Common.Tests/CommonSPDXUtilsTests.cs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Sbom.Parser;
+
+using Microsoft.Sbom.Common;
+using Microsoft.Sbom.Common.Utils;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+[TestClass]
+public class CommonSPDXUtilsTests
+{
+    [TestMethod]
+    public void GenerateSpdxPackageIdTest()
+    {
+        var spdxId = CommonSPDXUtils.GenerateSpdxPackageId("test");
+        Assert.IsTrue(spdxId.Contains(Constants.SPDXRefPackage));
```

Comment:
General note about Assert.IsTrue and Assert.IsNotTrue: If your unit test fails, the logs provide no hints to help you know why it failed. Methods like Assert.AreEqual or Assert.AreNotEqual are more informative, since they log the strings in question. If you have a scenario where you really need a boolean check, it's a good idea to use the description parameter to provide more info about the context. In this case, I'd use AreEqual, but let's assume for the sake of argument that the output was different each time--you could still glean more info with something like this:
```
Assert.IsTrue(spdxId.Contains("blah"), $"spdxId: {spdxId}");
```